functions-methods.js

```diff
@@ -20,7 +27,19 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail (email){
+    if (email.split("@")[1].includes("novi-education.nl"))
```

Comment:
Keurig! Het is wel netter en overzichtelijk om bijvoorbeeld email.split("@")[1].includes("novi-education.nl") in een variabele te zetten, vooral wanneer je deze vaker gaat gebruiken.